pkgs/development/python-modules/coordinates/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildPythonPackage, fetchFromGitHub, pythonOlder
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "coordinates";
+  version = "0.4.0";
+
+  disabled = pythonOlder "3.5";
+
+  src = fetchFromGitHub {
+    owner = "clbarnes";
+    repo = "coordinates";
+    rev = "v${version}";
+    sha256 = "1zha594rshjg3qjq9mrai2hfldya282ihasp2i3km7b2j4gjdw2b";
+  };
+
+  checkInputs = [ pytest ];
+
+  checkPhase = ''
+    pytest tests/
```

Comment:
should be (not ideal admittedly)

checkPhase = ''
  runHook preCheck
  pytest tests/
  runHook postCheck
'';

Reply:
in most cases, the explicit hooks aren't super necessary